src/fzx/tui/term_app.cpp

```diff
@@ -63,11 +64,24 @@ void TermApp::processTTY()
       continue;
     }
     if (*key == kEnter || *key == kCtrlU) {
+      if (*key == kEnter) {
```

Comment:
Can be moved out of the surrounding if statement
```cpp
if (*key == kEnter) {
  quit();
  return;
} else if (*key == kCtrlU) {
  // ...
```

Reply:
By the way, this looks exactly the same as Ctrl-C, so maybe set some success flag here that can be checked before printing out the result